### DISCUSSION TITLE
Welcome to GPD-Win-Max-Hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by kingo132** [kingo132/GPD-Win-Max-Hackintosh#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dicker182** [comment]
Nice to meet you. Love that project! I am already using your EFI for Big Sur and it runs beautful.

Thanks for your work!

**Reply by kingo132** [reply]
Great, if you have any improvement, please update this EFI to let everyone use it.

**Comment by seamonkey420** [comment]
awesome project! am working on getting Big Sur on my Win Max and will post bac results. Thanks again for this project, i tried and failed prev getting opencore and big sur going and this def seems like it will be the way to go!

**Reply by kingo132** [reply]
Yes, I have tested big sur for a little while. It can boot into the desktop. However, I think you need to change AirportItlwm.kext to the big sur version in order to get the wifi to work.

**Comment by Azkali** [comment]
Hi, thanks for crediting me here, I really appreciate ! 

I was intending to do this project at first but didn't had a Win Max but I am getting one this week to replace my win 2 which died today ... 

Are you using discord by any chance so we can discuss about future update and upgrades ? I'd like to contribute and hopefully fix the remaining issue with HDMI. Great job by the way !

**Reply by kingo132** [reply]
> Hi, thanks for crediting me here, I really appreciate !
> 
> I was intending to do this project at first but didn't had a Win Max but I am getting one this week to replace my win 2 which died today ...
> 
> Are you using discord by any chance so we can discuss about future update and upgrades ? I'd like to contribute and hopefully fix the remaining issue with HDMI. Great job by the way !

That's great! I'm a newbie to GitHub and don't know how to let the project can be contributed by more than one person. Let me figure it out these days. If you are in China or you can use QQ. You can also add this QQ group: 437973496 to join our chat.

**Reply by kingo132** [reply]
I happen to see this discord app today, this is my discord account: kingo132#1331.

**Comment by Juntei** [comment]
Hi ! 
I follow the tutorial i've found on youtube to dualboot windows and macos ( https://www.youtube.com/watch?v=MeQ47uuKQ5k&t=608s ).
However when I boot on "Install macos catalina" it doesn't work, i've just got few sentences like you, but then it stops. 
Is it the original SSD (512GB) that isn't compatible with the installation of OS catalina ?
Thanks
